File: modul 3/PRAK305-2410817320004-AmandhaCitraMustika.py
total_detik = int(input())
hari = total_detik//86400
sisa_detik = total_detik %86400

jam = sisa_detik//3600
sisa_detik %=3600
menit = sisa_detik//60
detik = sisa_detik%60

if hari>0 :
    print(f"{int(hari)} hari {jam:02}:{menit:02}:{detik:02}")
else :
    print(f"{jam:02}:{menit:02}:{detik:02}")

File: modul 3/PRAK305-2410817320004-AmandhaCitraMustika.c
#include <stdio.h>

int main(){
int total, detik, menit, jam, hari;
scanf("%d", &total);

hari = total/86400; total %=86400;
jam = total/3600; total %=3600;
menit = total/60;
detik = total %60;

if(hari>0){
    printf("%d hari %02d:%02d:%02d", hari, jam, menit, detik);
}
else{
    printf("%02d:%02d:%02d", jam, menit, detik);
}
    return 0;
}